packages/eslint-config/package.json

```diff
@@ -1,20 +1,34 @@
 {
   "description": "Wire ESLint config",
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "2.13.0",
+    "@typescript-eslint/parser": "2.13.0",
+    "eslint": "6.8.0",
+    "eslint-config-prettier": "6.9.0",
+    "eslint-plugin-jsdoc": "18.6.2",
+    "eslint-plugin-no-unsanitized": "3.0.2",
+    "eslint-plugin-prettier": "3.1.2",
+    "eslint-plugin-react": "7.17.0",
+    "eslint-plugin-react-hooks": "2.3.0",
+    "eslint-plugin-sort-keys-fix": "1.1.0",
+    "prettier": "1.19.1"
+  },
   "license": "GPL-3.0",
   "main": "eslintrc.json",
   "name": "@wireapp/eslint-config",
   "peerDependencies": {
-    "eslint": ">=5",
-    "eslint-config-prettier": ">=4",
-    "eslint-plugin-import": ">=2",
-    "eslint-plugin-jsdoc": ">=4",
+    "@typescript-eslint/eslint-plugin": ">=2",
+    "@typescript-eslint/parser": ">=2",
+    "eslint": ">=6",
+    "eslint-config-prettier": ">=6",
+    "eslint-plugin-jsdoc": ">=18",
     "eslint-plugin-no-unsanitized": ">=3",
     "eslint-plugin-prettier": ">=3",
     "eslint-plugin-react": ">=7",
-    "eslint-plugin-simple-import-sort": ">=4",
+    "eslint-plugin-react-hooks": ">=2",
     "eslint-plugin-sort-keys-fix": ">=1",
-    "prettier": ">=1.17"
+    "prettier": ">=1.19"
   },
   "repository": "https://github.com/wireapp/wire-web-packages/tree/master/packages/eslint-config",
-  "version": "1.3.0"
+  "version": "1.3.0-beta.22"
```

Comment:
Let's remove the beta tag

Reply:
Will the major version increased automatically or we have to do something? I already have a BREAKING CHANGE in the description